src/csharp/Microsoft.Spark.Worker/SimpleWorker.cs

```diff
@@ -38,6 +40,13 @@ internal void Run(int port)
                 ISocketWrapper socket = SocketFactory.CreateSocket();
                 socket.Connect(IPAddress.Loopback, port, secret);
 
+                if (_version.Major == 3 && _version.Minor == 2)
```

Comment:
This is only going to target 3.2.x .  If spark 3.3 is released we'll have to revisit this.  Since this contract change merged into spark's master branch, we should probably do `>=` instead of `==`

Reply:
Done